validation/README.md

```diff
@@ -0,0 +1,114 @@
+
+## Validation processes
+
+1.  *order* - The order of execution of the validation steps may be important. Validation for some items may need to occur before other items.
+2.  *items* - There are several different items that require validation. Items include: tables, variables, and specific value range for example, validation for a range of dates.
+3.  *platform* - Validation may differ depending on the platform. For example, Excel requires implementation using Excel conditional rules, whereas ArcGIS, Tableau and other platforms allow validation using application-specific applications and also open languages.
+4.  *role* - there may be a specific role(s), purpose or timing of a validation. If so, it may be helpful to define the role. For example, a validation rule may only be meaningful for data entry. In this situation, the role can be identified.
+5.  *language* - The implementation process should support different programming languages if required. For example, the Excel templates will include data validation where possible. However, not all validation steps can be included in Excel, and data validation occurs in multiple steps. Therefore, all validation steps are also able to be executed in Python.
+6.  *description* - Each validation step should include a clear human-readable description in English and French.
+7.  *required* - Validation should indicate whether a variable is required.
+8.  *dependencies* - Whether there are dependencies on other variables or validation steps, in addition to the order of validation.
+9.  *warning* - The number and type of warnings should be presented in human-readable form. Warnings should be presented in different formats, platforms, programs, etc.
+10. *errors* - The number and type of errors should be presented in human-readable form. Errors should be presented in different formats, platforms, programs, etc.
+11. *normalization (transformation*) - if values can be transformed, then this should occur. For example, data submitted with a non-SI unit (e.g., liters/hour) could be converted to its SI equivalent (m3/s) Variable mapping should be considered during variable mapping and not during validation, despite recognizing that mapping could be performed using normalization methods.
+12. *coercion* - If values can be coerced, then this should happen. i.e. integer as text should be coerced to numbers and a warning issued that this occurred.
+13. *version-specific* - Validation should indicate what version of the ODM the step is applicable for. Provisions for coercion and/or transforming old versions variables, types, and coercion of values to the most recent version. Warnings and errors should be issued, as appropriate.
+14. *wide table* - Validation of the wide table should be supported. Derived variable names should be validated, if possible.
```

Comment:
First mention in the document of the wide table. Maybe a small reminder of what the wide table is could be added here.